### ISSUE TITLE
Skipping Notes

### ISSUE THREAD

**Issue by stkproduction11** [bui/taiko-web#265]
﻿###### 下記の問題を説明してください。 スクリーンショットと診断情報を含めてください。
###### Describe the problem you are having below. Please include a screenshot and the diagnostic information.

I have hit the notes and the scroll occasionally skips for no reason

**Comment by BlueJYT** [comment]
Can you be more specific?

**Comment by stkproduction11** [comment]
Let me rephrase that, the notes usually skips and scrolls mostly because of lag. Think you can fix the lags?

**Comment by BlueJYT** [comment]
I'm not a developer, but I usually try to help people, if you are lagging, make sure that your tabs are clear... seriously... this app takes A LOT out of your processers, so it would be the best playing with the optimal settings.
